content/docs/guides/branch-archiving.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Branch archiving
+subtitle: Learn how Neon automatically archives inactive branches to cost-effective storage
+enableTableOfContents: true
+updatedOn: '2024-10-15T17:19:50.116Z'
+---
+
+<Admonition type="note" title="Only enabled on the Free Plan">
+Automatic archiving of inactive branches is only enabled on the Free Plan. Branch archiving will be introduced on paid plans at a later date.
+</Admonition>
+
+To minimize storage costs, Neon automatically archives branches that are:
+
+- Older than **14 days**.
+- Have not been accessed for the past **24 hours**
+
+Both conditions must be true for a branch to be archived.
+
+## Unarchiving a branch
+
+**No action is required to unarchive a branch. It happens automatically.**
+
+Connecting to an archived branch, querying it, or performing some other action that accesses it will trigger the unarchive process. Connection times and query times may be slower while a branch is unarchived.
+
+The following actions will automatically unarchive a branch, transferring the branch's data back to regular Neon storage:
+
+- Connecting to or querying the branch from a client or application
+- Querying the branch from the Neon SQL Editor
+- Viewing the branch on the Tables page in the Neon Console
+- Creating a child branch
+- Creating a role on a branch
+- Creating a database on a branch
+- Reset the branch from its parent
+- Performing a restore operation on a branch
+- Setting the branch as protected
+- Running Neon CLI commands and API calls that access the branch
```

Comment:
can we add links to relevant docs page for these actions?

Reply:
added